forms-flow-web/src/components/Bundle/bundleDesign/BundleCreateAndEdit.js

```diff
@@ -133,6 +146,12 @@ const BundleCreate = ({ mode }) => {
           formData.name = addTenantkey(formData.name, tenantKey);
         }
       }
+  
+      if (!bundleDescription.trim()){
+        toast.error( "Bundle Description is Required");
+         throw "Description is empty";
```

Comment:
you can write this condition as the first line of the function and then return. if you return anything the remaining code will not execute

Reply:
yes